app/assets/javascripts/custom.js

```diff
@@ -52,15 +67,17 @@ $(document).ready(function(){
         },
         order_details: {
           sub_total: line_total,
-          total: _total
+          total: _total,
+          prep_time: line_prep_total,
```

Comment:
Identifier 'prep_time' is not in camel case.<br>Identifier 'line_prep_total' is not in camel case.